main.go

```diff
@@ -70,6 +76,40 @@ func main() {
 	}
 }
 
+//abHandler ... percentA is the percentage of request that handler 'a' is used
+func abHandler(a, b http.Handler, percentA int) http.Handler {
+	if percentA < 0 || percentA > 100 {
+		panic("Percent 'a' but be between 0 and 100")
```

Comment:
it panics on startup when `abHandler` is called during route setup, but I've added a check when reading in the config value so it should never happen

i'll leave the panic here in case it gets used incorrectly somewhere else in future